fluss-client/src/main/java/com/alibaba/fluss/client/admin/Admin.java

```diff
@@ -51,6 +52,10 @@
  */
 @PublicEvolving
 public interface Admin extends AutoCloseable {
+
+    /** Get the current server node information. asynchronously. */
+    CompletableFuture<List<ServerNode>> getServerNodes();
+
```

Comment:
Change the ServerNode's class comment as: 
```
/**
 * Information about a Fluss server node.
 *
 * @since 0.6
 */
@PublicEvolving
public class ServerNode {
```